src/main/node/spec/appSpec.js

```diff
@@ -0,0 +1,66 @@
+const request = require('supertest');
+
+describe('app', () => {
+    const makeApp = require('../lib/app');
+    let app;
+
+    beforeEach(() => {
+        app = makeApp(x => x ** 2);
```

Comment:
I think we need a more complex function to really assert the difference in return types, what do you think?
plainText(9) is the same as json(9)